lib/engine/action/switch_trains.rb

```diff
@@ -14,7 +14,11 @@ def initialize(entity, slots: nil)
 
       def self.h_to_args(h, _game)
         {
-          slots: h['slots']&.map { |m| m.to_i },
+          slots: h['slots']&.map do |m|
+            Integer(m)
+          rescue StandardError
```

Comment:
what is this

Reply:
The 1873 implementations needs an int but I have a string Token and thats the only way in ruby tontry parse an int.

I could of course create a new action but I thought I reuse as much as possible

Reply:
I think I'm creating a new view and a new action. The more I think about it the less I like it.

Reply:
ok so you will redo this?

Reply:
New view + Action is available